src/plasmapy/diagnostics/charged_particle_radiography/synthetic_radiography.py

```diff
@@ -198,7 +197,9 @@ def __init__(
         detector: u.Quantity[u.m],
         dt=None,
         dt_range=None,
-        field_weighting="volume averaged",
+        field_weighting: Literal[
+            "volume averaged", "nearest neighbor"
+        ] = "volume averaged",
```

Comment:
📝 `typing.Literal` is pretty helpful since it lets us show that there are only certain values that should be accepted as a particular argument. This line means that `field_weighting` can only be either `"volume_averaged"` or `"nearest neighbor"`.